linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -146,7 +146,7 @@ impl ClientWrapper {
             .current_dir(self.path_provider.path())
             .env(
                 "RUST_LOG",
-                std::env::var("RUST_LOG").unwrap_or(String::from("linera=default")),
+                std::env::var("RUST_LOG").unwrap_or(String::from("linera=debug")),
```

Comment:
@Twey: `default` is not a log level. This is probably what you meant [here](https://github.com/linera-io/linera-protocol/pull/2257/files#r1686167112)?

Reply:
Debug would be very verbose. Something about JSON vs text ?

Reply:
Maybe. What do you mean with JSON vs. text?
Anyway, I'll revert this for now, since it's unrelated to this PR.